structuredheadings/plugin.js

```diff
@@ -301,22 +317,23 @@
       setupElements(editor);
       setupCommands(editor);
       setupStyles(editor);
+      self.currentScheme = editor.config.autonumberDefaultScheme;
 
       editor.on("dataReady", function () {
         self.currentScheme = self.detectScheme();
       });
 
       //Format Dropdown
       editor.ui.addRichCombo("NumFormats", {
-        label: "Formats",
-        title: "Numbering Formats",
+        label: "Formatting",
+        title: "Text Formatting",
```

Comment:
Text formatting is the left one (p, h2-h5, etc) not the right one 

this was in https://docs.google.com/document/d/1KOKwdgYA8_bzHAnqCx1jhzP-v7SlM5d7vj_-I0yiAYQ/edit